compiler/quilt/tools/command.py

```diff
@@ -497,6 +497,14 @@ def build(package, path=None, dry_run=False, env='default'):
     """
     # TODO: rename 'path' param to 'target'?
     team, _, _ = parse_package(package)
+    logged_in_team = _find_logged_in_team()
+    if logged_in_team is not None and team is None:
+        answer = input("You're logged in as a team member, but you aren't specifying " +
+                        "a team for the package you're currently building. Did you mean " +
+                        "quilt build {team}:{package}? N to continue. (Y/n)".format(
```

Comment:
I feel like "y" to continue is the convention; "n" feels weird to be. Also, should probably add a "force" parameter to avoid the prompt.

Reply:
The prompt only appears if the install fails. I will add a force parameter to API, but I think CLI is fine.

Reply:
How would you phrase it to make 'y' the default?

Reply:
Something like "Are you sure you want to continue?"